### DISCUSSION TITLE
大佬你的openclash的uci-default设置是怎么写的啊？

### DISCUSSION THREAD

**Discussion by m0eak** [chenmozhijin/OpenWrt-K#66] [Q&A] [answered]
别的luci app保存了都会显示未保存的内容，openclash保存了没有，而且更改设置后uci changes openclash也不返回更改内容

**Comment by m0eak** [comment]
还有我加入了autocore之后首页没有cpu使用率和温度，是不是还少了什么操作啊

**Comment by chenmozhijin** [comment, marked as answer]
> 别的luci app保存了都会显示未保存的内容，openclash保存了没有，而且更改设置后uci changes openclash也不返回更改内容

命令行执行
```
uci show openclash
```
就能看到openclash的配置了

> 还有我加入了autocore之后首页没有cpu使用率和温度，是不是还少了什么操作啊

immortalwrt在首页显示cpu使用率和温度好像跟autocore没有什么关系
我看了一下immortalwrt修改了[luci/modules/luci-base/root/usr/share/rpcd/ucode/luci](https://github.com/immortalwrt/luci/blob/abcfaaf79e9ba8945e9263b7e0fd3c593c375eb8/modules/luci-base/root/usr/share/rpcd/ucode/luci#L585C1-L643C4)
[luci/modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/10_system.js](https://github.com/immortalwrt/luci/blob/f20b66a990d93533ebce99db59cab541e3ed4b9f/modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/10_system.js)
[luci/modules/luci-mod-status/root/usr/share/rpcd/acl.d/luci-mod-status.json](https://github.com/immortalwrt/luci/blob/f20b66a990d93533ebce99db59cab541e3ed4b9f/modules/luci-mod-status/root/usr/share/rpcd/acl.d/luci-mod-status.json)
可能还有其他的地方我没有找到的
